src/ontology/uberon.Makefile

```diff
@@ -934,40 +934,59 @@ subsets/life-stages-core.owl: uberon.owl
 # Taxon subsets
 # ----------------------------------------
 
-all_taxmods: $(TAXMODSDIR)/uberon-taxmod-amniote.obo $(TAXMODSDIR)/uberon-taxmod-euarchontoglires.obo
-
-$(TAXMODSDIR)/uberon-taxmod-euarchontoglires.owl: $(TMPDIR)/uberon-taxmod-314146.owl
-	cp $< $@
-
-$(TAXMODSDIR)/uberon-taxmod-amniote.owl: $(TMPDIR)/uberon-taxmod-32524.owl
-	cp $< $@
-
-$(TAXMODSDIR)/uberon-taxmod-human.owl: $(TMPDIR)/uberon-taxmod-9606.owl
-	cp $< $@
-
-subsets/%-basic.owl: $(TAXMODSDIR)/uberon-taxmod-%.owl tmp/simple-slim-seed.txt
-	$(ROBOT) reason --input $< \
-		        --reasoner ELK --equivalent-classes-allowed all --exclude-tautologies structural \
+TAXON_ID_human            = NCBITaxon:9606
+TAXON_ID_mouse            = NCBITaxon:10090
+TAXON_ID_xenopus          = NCBITaxon:8353
+TAXON_ID_drosophila       = NCBITaxon:7227
+TAXON_ID_gnathostome      = NCBITaxon:7776
+TAXON_ID_amniote          = NCBITaxon:32524
+TAXON_ID_euarchontoglires = NCBITaxon:314146
+TAXON_ID_nematode         = NCBITaxon:6237
+TAXON_ID_zebrafish        = NCBITaxon:7955
+
+# Strategy to use to create the taxon subsets:
+# - default: OWLTools' original strategy; given a root class R and a
+#            taxon T, assert 'R SubClassOf: in_taxon some T' and exclude
+#            all classes that are unsatisfiable because of that
+#            assertion.
+# - precise: alternative strategy; given a root class R and a taxon T,
+#            iterate through all subclasses C of R and include classes
+#            for which the expression 'C and in_taxon some T' is
+#            satisfiable; about 3-5 times slower than 'default'.
+TAXON_SUBSET_STRATEGY = default
+
+TAXON_SUBSET_ROOTS = UBERON:0001062 UBERON:0000000
```

Comment:
Does this second root exist? I don't think I have ever seen it

Reply:
Yes it does. It’s '[processual entity](http://purl.obolibrary.org/obo/UBERON_0000000)', which is the parent of '[life cycle](http://purl.obolibrary.org/obo/UBERON_0000104)' and all the '[life cycle stage](http://purl.obolibrary.org/obo/UBERON_0000105)' terms.